README.md

```diff
@@ -70,6 +49,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT.md) licensed.
```

Comment:
### Your MIT.md File
![MIT2](https://user-images.githubusercontent.com/93039359/198332260-d41754c6-a897-4e5a-825e-90faeee1b19f.PNG)


- [ ] Its great that you have added an link to your MIT.md file and it is working 😊 But as you can see that your MIT.md file is looking like a template. Kindly add your name, current year and app-type in your MIT.md file to make your project licensed 👍